app/facades/food_facade.rb

```diff
@@ -0,0 +1,9 @@
+class FoodFacade
+  class << self 
+    def search_foods(food)
+
+      data = FoodService.call_for_a_food(food)
+      FoodIndex.new(data)
```

Comment:
Would have been a good idea to extract the total_hits from the instantiation of your `Food` objects here. `FoodIndex` is not SRP.

Reply:
One thing to consider would be to return a 2d array from the facade where the structure looks like this: 
`[<num_of_total_hits>, [FoodObject1, FoodObject2...]]`
Then you could unpack this in your controller like so: 
`@total_hits, @top_10_foods = FoodFacade.search_foods(food)`